### PR TITLE
feat: entry points plus

### PR DESCRIPTION
**Pull request by capJavert**
## Changes

- add new soon modals
- connect them to entry points
  - feed settings under content types & languages
  - custom feeds on click on "new feed"
  - bookmarks on click on new bookmark

Engineering decision, for each of these blocks:
image

If you are a plus member I just hide them (because you still are waiting for the feature but you are already a plus member. 

## Events

Did you introduce any new tracking events?

| Type   | event_name  | value |
|--------|-------------|-------|
| Change| upgrade subscription | target_id: advanced custom feed soon modal |
| Change| upgrade subscription | target_id: bookmark folder soon modal |
| Change| upgrade subscription | target_id: feed settings |

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

AS-793

### Preview domain
https://as-793-entry-points-plus.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/components/sidebar/sections/CustomFeedSection.tsx [hunk 1]
```diff
@@ -39,9 +43,24 @@ export const CustomFeedSection = ({
...
         path: `${webappUrl}feeds/new`,
         requiresLogin: true,
+        isForcedClickable: true,
+        action: (
+          event: React.MouseEvent<HTMLAnchorElement | HTMLButtonElement>,
+        ) => {
+          if (showPlusSubscription) {
+            event.preventDefault();
+
+            openModal({ type: LazyModal.AdvancedCustomFeedSoon, props: {} });
```
Comment (rebelchris) [inline packages/shared/src/components/sidebar/sections/CustomFeedSection.tsx:53]: Also here, props not needed I think?

File: packages/shared/src/components/modals/soon/BookmarkFolderSoonModal.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+              Upgrade to daily.dev Plus today, get an early adopter discount and
+              be among the first to experience it as soon as it launches!
+            </Typography>
+            <Button
+              className="w-full"
+              tag="a"
+              type="button"
+              variant={ButtonVariant.Primary}
+              size={ButtonSize.Medium}
+              href="/plus"
```
Comment (rebelchris) [inline packages/shared/src/components/modals/soon/BookmarkFolderSoonModal.tsx:75]: Better to use webappUrl here I think (for extension)

File: packages/shared/src/components/modals/soon/AdvancedCustomFeedSoonModal.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+              Upgrade to daily.dev Plus today, get an early adopter discount and
+              be among the first to experience it as soon as it launches!
+            </Typography>
+            <Button
+              className="w-full"
+              tag="a"
+              type="button"
+              variant={ButtonVariant.Primary}
+              size={ButtonSize.Medium}
+              href="/plus"
```
Comment (rebelchris) [inline packages/shared/src/components/modals/soon/AdvancedCustomFeedSoonModal.tsx:76]: Use webapp url here as well

File: packages/shared/src/components/filters/ContentTypesFilter.tsx [hunk 4]
```diff
@@ -13,18 +13,102 @@ import { Divider } from '../utilities';
...
+                  and be among the first to experience it as soon as it
+                  launches!
+                </Typography>
+                <Button
+                  className="flex"
+                  tag="a"
+                  type="button"
+                  variant={ButtonVariant.Primary}
+                  size={ButtonSize.Medium}
+                  href="/plus"
```
Comment (rebelchris) [inline packages/shared/src/components/filters/ContentTypesFilter.tsx:89]: And here :)

**Review by rebelchris — CHANGES_REQUESTED** [review]
Nice minor full URL use, but otherwise looks great!

**Review by rebelchris — APPROVED** [review]
I just asked @idoshamun  to verify the target_id's but think it's fine.

File: packages/shared/src/lib/log.ts [hunk 5]
```diff
@@ -287,6 +287,9 @@ export enum TargetId {
   PlusBadge = 'plus badge',
   Onboarding = 'onboarding',
   BlockedWords = 'block words',
+  AdvancedCustomFeedSoonModal = 'advanced custom feed soon modal',
```
Comment (idoshamun) [inline packages/shared/src/lib/log.ts:290]:
```suggestion
  CustomFeed = 'custom feed',
```

File: packages/shared/src/lib/log.ts [hunk 6]
```diff
@@ -287,6 +287,9 @@ export enum TargetId {
...
+  BookmarkFolderSoonModal = 'bookmark folder soon modal',
```
Comment (idoshamun) [inline packages/shared/src/lib/log.ts:291]:
```suggestion
  BookmarkFolder = 'bookmark folder',
```

**Review by idoshamun — APPROVED** [review]

**Pull request merged**